tensorboard/plugins/interactive_inference/witwidget/notebook/visualization.py

```diff
@@ -34,16 +35,24 @@ def __init__(self, examples):
     """Constructs the WitConfigBuilder object.
 
     Args:
-      examples: A list of tf.Example or tf.SequenceExample proto objects.
-      These are the examples that will be displayed in WIT. If not model to
+      examples: A list of tf.Example or tf.SequenceExample proto objects, or
+      raw JSON objects. JSON is allowed only for CMLE-hosted models (see
+      'set_cmle_model' and 'set_compare_cmle_model methods).
+      These are the examples that will be displayed in WIT. If no model to
       infer these examples with is specified through the methods on this class,
       then WIT will display the examples for exploration, but no model inference
       will be performed by the tool.
     """
     self.config = {}
-    self.set_examples(examples)
     self.set_model_type('classification')
     self.set_label_vocab([])
+    if len(examples) > 0 and not (
```

Comment:
It looks like WIT infers model's input type from the type of examples (json vs tf.Example). I think it would be good to add an extra force_json_conversion flag in set_cmle_model to enable comparison of two models where one accepts tf.Example and other json as tf.Example->json conversion is possible. json->tf.Example conversion is not doable in all cases since the data types are missing in json so we cannot make this process fully transparent hence the requirement for force_json_conversion flag.

Reply:
done